### PR TITLE
Muscle 3.8.31 (enable RHEL Linux support, does not affect Mac OS X)

### PR DESCRIPTION
**Pull request by tseemann**
The Makefile for Muscle attempts to create a statically linked binary (libstdc++) only for Linux.

This will fail in the general case on modern Linux. There is no need for it to be static. The Mac version simply links dynamically. Best to be consistent.

### CODE REVIEW

File: muscle.rb [hunk 1]
```diff
@@ -22,6 +22,13 @@ def install
               "#include <mach/task_info.h>",
               "#include <mach/vm_statistics.h>\n#include <mach/task_info.h>"
 
+    # This patch makes 3.8.31 build on RHEL 7.x
+    # It ONLY affects Linux (in an "if Linux" clause in the 'mk' script)
+    # It is unnecessary to create a static binary
```
Comment (sjackman) [inline muscle.rb]: Please report in a comment the error message that you're fixing.

Comment (tseemann) [inline muscle.rb]: @sjackman ok, added.

Comment (sjackman) [inline muscle.rb]: Hmm. I don't see it. Did you push it?

Comment (tseemann) [inline muscle.rb]: Sorry I thoguht you meant a comment in the code, here is the Linux error message:

```
collect2: error: ld returned 1 exit status
strip: 'muscle': No such file
ls: cannot access muscle: No such file or directory
sum: muscle: No such file or directory
make: *** [muscle] Error 1
```

**Comment by tseemann** [comment]
@sjackman Any chance this one can be accepted now?   So many things depend on it.

**Comment by sjackman** [comment]
Done. Thanks for your contribution, Torsten.

**Pull request closed without merging**